frontend/src/API/ImagesAPI.ts

```diff
@@ -21,7 +21,27 @@ export default class ImagesAPI {
 
   public static uploadMemberImage(body: Blob): Promise<void> {
     return this.getSignedURL().then((url) => {
-      APIWrapper.put(url, body).then((res) => res.data);
+      const headers = { contentType: 'image/jpeg' };
```

Comment:
I think changing `contentType` to be `'Content-Type'` should fix the CORS issues

Reply:
yes, it worked! thank you!